cmds/core/kexec/kexec_linux.go

```diff
@@ -90,9 +93,37 @@ func main() {
 		purgatory.Debug = log.Printf
 	}
 
-	if (!opts.exec && flag.NArg() == 0) || flag.NArg() > 1 {
+	var kernelpath string
+	if flag.NArg() > 0 {
+		kernelpath = flag.Arg(0)
+	}
+	// Option values from linux image config takes precendence
```

Comment:
precedence I think

Reply:
Done.

I'll look if we can add a spell checker